src/Umbraco.Web/RelatedLinksTypeConverter.cs

```diff
@@ -7,25 +7,12 @@
 using Newtonsoft.Json.Linq;
 
 using Umbraco.Core;
-using Umbraco.Core.Logging;
 using Umbraco.Web.Models;
 
 namespace Umbraco.Web
 {
     public class RelatedLinksTypeConverter : TypeConverter
     {
-        private readonly UmbracoHelper _umbracoHelper;
-
-        public RelatedLinksTypeConverter(UmbracoHelper umbracoHelper)
```

Comment:
I've removed this constructor, as we don't want to capture an `UmbracoContext` (as that's request based).